commands/delete-key.js

```diff
@@ -20,6 +21,24 @@ async function deleteAccessKey(options) {
     console.log(`Deleting key = ${options.accessKey} on ${options.accountId}.`);
     const near = await connect(options);
     const account = await near.account(options.accountId);
-    const result = await account.deleteKey(options.accessKey);
-    inspectResponse.prettyPrintResponse(result, options);
+    let accessKeys = await account.getAccessKeys();
+    let fullAccessKeys = accessKeys.filter(accessKey => accessKey.access_key.permission === 'FullAccess');
```

Comment:
Looks like we will not delete the key if this condition is not met.
It's a critical piece of code. Let's work on it together. Here is a thought about warnings that we should have here:
https://github.com/near/near-cli/issues/834#issuecomment-968970448